doc/CHANGES-staging/func_groupcount_group_variables.txt

```diff
@@ -0,0 +1,47 @@
+Subject: func_groupcount
```

Comment:
The CHANGES file is no longer needed. This should go in UserNote: in the commit message (and probably be condensed, usage explanations should not be in the commit message, but in the XML documentation).